### PR TITLE
[GCP-7696] Storage: generate_signed_url API documentation misleading for response_type/disposition

### PR DESCRIPTION
**Pull request by HemangChothani**
Fixes [#7696](https://github.com/googleapis/google-cloud-python/issues/7696)

### CODE REVIEW

File: storage/google/cloud/storage/_signing.py [hunk 1]
```diff
@@ -321,8 +321,7 @@ def generate_signed_url_v2(
 
     :type response_type: str
     :param response_type: (Optional) Content type of responses to requests for
-                          the signed URL. Used to over-ride the content type of
-                          the underlying resource.
+                          the signed URL.Ignored if content_type is set.
```
Comment (Emar-Kar) [inline storage/google/cloud/storage/_signing.py]: Whitespace is missing between the sentences.

File: storage/google/cloud/storage/blob.py [hunk 2]
```diff
@@ -365,8 +365,7 @@ def generate_signed_url(
 
         :type response_type: str
         :param response_type: (Optional) Content type of responses to requests
-                              for the signed URL. Used to over-ride the content
-                              type of the underlying blob/object.
+                              for the signed URL.Ignored if content_type is set.
```
Comment (mf2199) [inline storage/google/cloud/storage/blob.py]: Same here.

File: storage/google/cloud/storage/_signing.py [hunk 3]
```diff
@@ -321,8 +321,7 @@ def generate_signed_url_v2(
 
     :type response_type: str
     :param response_type: (Optional) Content type of responses to requests for
-                          the signed URL. Used to over-ride the content type of
-                          the underlying resource.
+                          the signed URL. Ignored if content_type is set.
```
Comment (sumit-ql) [inline storage/google/cloud/storage/_signing.py]: Minor :
the signed URL. Ignored if content_type is set on object/blob metadata.

File: storage/google/cloud/storage/_signing.py [hunk 4]
```diff
@@ -467,8 +466,7 @@ def generate_signed_url_v4(
...
```
Comment (sumit-ql) [inline storage/google/cloud/storage/_signing.py]: Same as above.

File: storage/google/cloud/storage/blob.py [hunk 5]
```diff
@@ -365,8 +365,7 @@ def generate_signed_url(
 
         :type response_type: str
         :param response_type: (Optional) Content type of responses to requests
-                              for the signed URL. Used to over-ride the content
-                              type of the underlying blob/object.
+                              for the signed URL. Ignored if content_type is set.
```
Comment (sumit-ql) [inline storage/google/cloud/storage/blob.py]: Same as above.

**Comment by mf2199** [comment]
As we've reached a consensus, it's time to close this one and open a public PR for that matter.

**Pull request closed without merging**